nuxeo-core/nuxeo-core-api/src/main/java/org/nuxeo/ecm/core/api/model/impl/AbstractProperty.java

```diff
@@ -428,6 +436,61 @@ public Property resolvePath(Path path) throws PropertyNotFoundException {
         return property;
     }
 
+    protected Property wrapDeprecatedPropertyIfNeeded(Property property) {
```

Comment:
The goal of `@since` is mainly to provide information for readers of the Javadoc. There's no Javadoc generated for protected methods, so `@since` is not very useful there.